apps/payments/next/app/layout.tsx

```diff
@@ -22,7 +22,7 @@ export default function RootLayout({
           data-testid="header"
         >
           <Image
-            src={firefoxLogo}
+            src={mozillaLogo}
             alt="Firefox logo"
```

Comment:
L26 replace `Firefox logo` with `Mozilla logo`.

Additionally, L7 and L8 replace `Firefox` with `Mozilla`.

Edit: for L7 and L8, should it be `account` or `accounts`? According to the branding guidelines:

> - "accounts" can and should be localized, "Mozilla" must be treated as a brand. Plural "Mozilla accounts" is used when referring to something affecting all Mozilla accounts, not just the individual's account.
> - "accounts" should be lowercase in almost all cases. Uppercase is reserved for special use cases where headline case is necessary, for example legal document names and references.

So I wasn't quite sure.

Reply:
For L7 and L8. Can these just go away? It doesn't seem as though they are ever referenced...

Reply:
Typically it says the same. If makes sense to 'your Mozilla account', then it's singlera

Reply:
I've updated the alt text btw.

Reply:
Good catch @dschom . I didn't even realize they weren't being used. Keep for now because I'm not sure if they are planning to be used and I'll check with the rest of SP. If necessary, we'll remove on our end 👍 . Thank you for making the change anyway! :mozheart: